tests/vmi_multus_test.go

```diff
@@ -1065,6 +1072,18 @@ func checkMacAddress(vmi *v1.VirtualMachineInstance, interfaceName, macAddress s
 	Expect(err).ToNot(HaveOccurred(), "MAC %q was not found in the VMI %s within the given timeout", macAddress, vmi.Name)
 }
 
+func setUpInterface(vmi *v1.VirtualMachineInstance, interfaceName string) error {
```

Comment:
the two functions are different only in the command aint they?
(talking about the Expect block), 
worth maybe a wrapper ?